packages/fxa-admin-server/src/gql/model/relying-party.model.ts

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { Field, ID, ObjectType } from '@nestjs/graphql';
+
+@ObjectType()
+export class RelyingParty {
+  @Field((type) => ID)
+  id!: string;
+
+  @Field()
+  name!: string;
+
+  @Field()
+  imageUri!: string;
+
+  @Field()
+  redirectUri!: string;
+
+  @Field()
+  canGrant!: boolean;
+
+  @Field()
+  publicClient!: boolean;
+
+  @Field()
+  createdAt!: number;
+
+  @Field()
+  trusted!: boolean;
+
+  @Field({ nullable: true })
```

Comment:
In fxa-shared/db/models/auth/relying-party.ts, this value is marked as non nullable. Just double checking there's not a mismatch here.

Reply:
Another good catch - this is definitely nullable, I'll change it over there.